client/components/screen-options-tab/index.js

```diff
@@ -94,7 +97,9 @@ const ScreenOptionsTab = ( { wpAdminPath } ) => {
 	return (
 		<div className="screen-options-tab" ref={ ref } data-testid="screen-options-tab">
 			<button className="screen-options-tab__button" onClick={ handleToggle }>
-				<span className="screen-options-tab__label">{ __( 'Screen Options' ) }</span>
+				<span className="screen-options-tab__label">
+					{ _x( 'View', 'View options to switch between' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'View', { context: 'Stats: List item action to view content'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/view-switch-tracking)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?